java/src/main/java/ai/rapids/cudf/Cuda.java

```diff
@@ -244,6 +244,27 @@ public synchronized void close() {
     }
   }
 
+  /**
+   * Gets the CUDA compute mode of the current device.
+   *
+   * @return the enum value of CudaComputeMode
+   */
+  public static CudaComputeMode getComputeMode() {
+    int nativeMode = Cuda.getNativeComputeMode();
+    switch (nativeMode) {
```

Comment:
Generally for other enums we put the mapping from native in the enum class itself. That way all of the info is in a single file and is consistent.  We also can make the code a lot smaller if we don't worry about performance. Because this is not something that is going to be called in a tight loop where performance matters, I would much rather have smaller code with guaranteed consistency.  You can use the `BinaryOp.fromNative` as an example.

```
  static BinaryOp fromNative(int nativeId) {
    for (BinaryOp type : OPS) {
      if (type.nativeId == nativeId) {
        return type;
      }
    }
    throw new IllegalArgumentException("Could not translate " + nativeId + " into a BinaryOp");
  }
```

Reply:
I changed the way of native mapping. For now, it follows above style.